Homepage implementada

ireneGarciaFernandez opened the pull request:
Ahora ya creo que se hace el merge en frontend, perdón XD

ireneGarciaFernandez committed:
Homepage implementada

lwangwan committed:
Merge branch 'frontend' into Homepage

rchacoma8 commented on frontend/src/router/index.js:
```diff
@@ -1,5 +1,6 @@
 import Router from 'vue-router'
+import Homepage from '@/components/Homepage'
 import Motos from '@/components/Motos.vue'
 import HelloWorld from '@/components/HelloWorld'
```
Esta línea da conflicto al runear el frontend. Hay que quitarla

rchacoma8 committed:
Update index.js

He quitado la línea import HelloWorld from '@/components/HelloWorld'
No me permitía hacer el run al no usarse ese import

rchacoma8 reviewed (commented):
Diseño muy bonito, logo muy apropiado, y se ve bien en modo móvil.
(El segundo logo parece que sea de mudanza, tho x) ) pero todo muy bien

ireneGarciaFernandez commented:
> 
> 
> Diseño muy bonito, logo muy apropiado, y se ve bien en modo móvil.
> (El segundo logo parece que sea de mudanza, tho x) ) pero todo muy bien

Te refieres al icon de building? al encabezado de las cards? ese puedo mirar alguno más adecuado pero como queria hacer una sección de información de la empresa pues era el que más se acercaba, voy a ver si hay icons de motos o algun vehiculo~

rchacoma8 commented:
> > Diseño muy bonito, logo muy apropiado, y se ve bien en modo móvil.
> > (El segundo logo parece que sea de mudanza, tho x) ) pero todo muy bien
> 
> Te refieres al icon de building? al encabezado de las cards? ese puedo mirar alguno más adecuado pero como queria hacer una sección de información de la empresa pues era el que más se acercaba, voy a ver si hay icons de motos o algun vehiculo~

No, no. No te preocupes, es que me había hecho gracia, like "mooving" y que también podía ser de mudanzas x) pero me parece correcto! Bien pensado
